tests/components/esphome/test_config_flow.py

```diff
@@ -1239,7 +1278,7 @@ async def test_reauth_fixed_via_dashboard_add_encryption_remove_password(
 ) -> None:
     """Test reauth fixed automatically via dashboard with password removed."""
     mock_client.device_info.side_effect = (
-        InvalidAuthAPIError,
+        InvalidEncryptionKeyAPIError("Wrong key", "test"),
```

Comment:
This has always been wrong. `InvalidAuthAPIError` is for password auth. `InvalidEncryptionKeyAPIError` is for noise/encrypt key wrong